ocs_ci/deployment/deployment.py

```diff
@@ -366,12 +393,59 @@ def subscribe_ocs(self):
         if subscription_plan_approval == "Manual":
             wait_for_install_plan_and_approve(self.namespace)
 
+    def get_zone_label(self):
```

Comment:
We also need a function to add zone label in case of vmware deployment

Reply:
I am expecting that should be done exactly after OCP deployment, not in b/w OCS deployments. Moreover, there should be some pattern to add zone labels.